src/views/Arys/ActionButtons/AryCopyModal/index.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useState, useCallback } from 'react';
+import { connect } from 'react-redux';
+
+import Modal from '#rscv/Modal';
+import ModalHeader from '#rscv/Modal/Header';
+import ModalBody from '#rscv/Modal/Body';
+import ModalFooter from '#rscv/Modal/Footer';
+import ChecklistInput from '#rsci/ChecklistInput';
+import SearchInput from '#rsci/SearchInput';
+
+import Button from '#rsu/../v2/Action/Button';
+
+import {
+    currentUserAdminProjectsSelector,
+} from '#redux';
+import {
+    RequestClient,
+    methods,
+} from '#request';
+
+import notify from '#notify';
+import _ts from '#ts';
+import _cs from '#cs';
+
+import {
+    notifyOnFailure,
+    notifyOnFatal,
+} from '#utils/requestNotify';
+
+import {
+    AddRequestProps,
+    Requests,
+    AppState,
+} from '#typings';
+
+import styles from './styles.scss';
+
+interface OwnProps {
+    className?: string;
+    requests: () => void;
```

Comment:
The typings for requests is not correct.

Reply:
@subinasr  I think we have a `Request` in `#typings'.
Refer to what is used in Questionnaire Builder.